Zend/zend_execute.h

```diff
@@ -207,6 +207,21 @@ static zend_always_inline zval* zend_assign_to_variable_ex(zval *variable_ptr, z
 	return variable_ptr;
 }
 
+static zend_always_inline void zend_safe_assign_to_variable_noref(zval *variable_ptr, zval *value) {
+	if (Z_REFCOUNTED_P(variable_ptr)) {
+		ZEND_ASSERT(Z_TYPE_P(variable_ptr) != IS_REFERENCE);
+		zend_refcounted *ref = Z_COUNTED_P(variable_ptr);
+		ZVAL_COPY_VALUE(variable_ptr, value);
+		if (!GC_DELREF(ref)) {
+			rc_dtor_func(ref);
+		} else {
+			gc_check_possible_root(ref);
```

Comment:
Use ``gc_check_possible_root_no_ref()``  or ``GC_DTOR_NO_REF()``

Reply:
Right, I don't know why I forgot this.